DataverseGen.Cli/example-dataversegen.config.json

```diff
@@ -6,7 +6,7 @@
   "TemplateName": "Main",
   "EnableConnectionStringValidation": true, //optional
   "ThrowOnEntityNotFound": true, //optional
-  "TemplateDirectory": "Templates",
+  "TemplateDirectory": "Templates", //optional
```

Comment:
it is not clear that we need to add a full path e.g. "Templates/EnelMedTemplate/" or only "EnelMedTemplate"